site/app/controllers/AuthenticationController.php

```diff
@@ -118,7 +118,7 @@ public function checkLogin() {
             }
             return false;
         }
-        $this->core->getAuthentication()->setUserId($_POST['user_id']);
+        $this->core->getAuthentication()->setUserId(trim($_POST['user_id'],' '));
```

Comment:
Let's make this change in https://github.com/Submitty/Submitty/blob/master/site/app/authentication/AbstractAuthentication.php#L32. You can also just do `trim()` with just one argument and leave the second to the optional default. (so just `trim(strtolower($user_id))`

Reply:
Thanks @MasterOdin , For reviewing my PR